caa-admin/recipes/default.rb

```diff
@@ -9,6 +15,7 @@
 
 execute 'caa_admin_down' do
   command 'svc -d /etc/service/caa-admin'
```

Comment:
I think it was necessary to stop it because cabal overwrites the existing binary https://github.com/metabrainz/chef-cookbooks/commit/e7d1ed9b58eec5edcb5d9af5af691b05dd3dc6f5

It's crappy, but I'll let it slide for now since it's not a critical service.